### PR TITLE
Make Swagger up to date

### PR DESCRIPTION
**Pull request by yellalena**

### CODE REVIEW

File: routes/applications.js [hunk 1]
```diff
@@ -2,30 +2,35 @@ const express = require("express");
...
 
 router.post("/applications", async (req, res) => {
-    const { name, text } = req.body;
+    const { name, about, email } = req.body;
     try {
         const result = await pool.query(
-            "INSERT INTO applications (name, about) VALUES ($1, $2) RETURNING *",
-            [name, text]
+            "INSERT INTO applications (name, about, email) VALUES ($1, $2, $3) RETURNING *",
+            [name, about, email]
```
Comment (marinacatbytes) [inline routes/applications.js:24]: can we change to insert the following fields:
id: number;
- name: string;
- email: string;
- about: string;
- languages: string[];
- photo: string; (optional)
- link to video: string; (optional)
- discord nickname: string;
- role: string;

File: scripts/db/applications.sql [hunk 2]
```diff
@@ -1,6 +1,7 @@
 CREATE TABLE applications (
     id SERIAL PRIMARY KEY, 
-    name VARCHAR(255),                  
+    name VARCHAR(255),
+    email VARCHAR(255) UNIQUE NOT NULL,
```
Comment (marinacatbytes) [inline scripts/db/applications.sql:4]: Need to adjust as per updated fields :)

**Review by marinacatbytes — DISMISSED** [review]
Thanks for this, could you just please amend the applications table fields

File: routes/applications.js (hunk 1 above)
Comment (yellalena) [reply routes/applications.js:24]: Yes, sure, but I think it should be done in a separate ticket: https://github.com/catbytes-community/webplatform-backend/issues/10
I can take it for the next week

File: scripts/db/applications.sql (hunk 2 above)
Comment (yellalena) [reply scripts/db/applications.sql:4]: will be done in https://github.com/catbytes-community/webplatform-backend/issues/10

**Review by marinakim44 — APPROVED** [review]

**Pull request merged**